pkg/gadget-service/api/api.proto

```diff
@@ -249,6 +249,8 @@ message GetGadgetInfoRequest {
   uint32 version = 3;
 
   uint32 flags = 4;
+
+  bool getExtraInfo = 5;
```

Comment:
I think we can drop the `get` and `extraInfo` is fine. Also please add a comment describing it, something like: